ui/webui/src/components/installation/InstallationProgress.scss

```diff
@@ -4,3 +4,13 @@
         text-align: left;
     }
 }
+
+.installation-progress-status {
+    &-success svg {
+        color: var(--pf-global--success-color--100);
```

Comment:
Interesting! What is the number at the end, opacity ? :)

Reply:
It's not oppacity, just different versions of red, green etc.

Reply:
I just checked what other icons of the types are using in PF4, with a quick grep in their repository:

```
[kkoukiou@sequioa patternfly]$ git grep success | grep icon--Color | grep global 
src/patternfly/components/Alert/alert.scss:  --pf-c-alert--m-success__icon--Color: var(--pf-global--success-color--100);
src/patternfly/components/HelperText/helper-text.scss:  --pf-c-helper-text--m-dynamic--m-success__item-icon--Color: var(--pf-global--success-color--100);
src/patternfly/components/Label/label.scss:  --pf-c-label--m-green__icon--Color: var(--pf-global--success-color--100);
src/patternfly/components/ModalBox/modal-box.scss:  --pf-c-modal-box--m-success__title-icon--Color: var(--pf-global--success-color--100);
src/patternfly/components/NotificationDrawer/notification-drawer.scss:  --pf-c-notification-drawer__list-item--m-success__list-item-header-icon--Color: var(--pf-global--success-color--100);
src/patternfly/components/Popover/popover.scss:  --pf-c-popover--m-success__title-icon--Color: var(--pf-global--success-color--100);
src/patternfly/components/Progress/progress.scss:  --pf-c-progress--m-success__status-icon--Color: var(--pf-global--success-color--100);
src/patternfly/components/ProgressStepper/progress-stepper.scss:    --pf-c-progress-stepper__step-icon--Color: var(--pf-global--success-color--100);
src/patternfly/components/Select/select.scss:  --pf-c-select--m-success__toggle-status-icon--Color: var(--pf-global--success-color--100);

```